Spine/Operation.swift

```diff
@@ -292,11 +282,11 @@ class SaveOperation: ConcurrentOperation {
 					self.state = .Finished
 					return
 				}
-				
 			} else {
 				if let data = responseData where data.length > 0 {
 					do {
-						try self.serializer.deserializeData(data, mappingTargets: [self.resource])
+						try self.serializer.deserializeData(data, mappingTargets: nil)
```

Comment:
Well, this one is just to test that deserialization throws no error. For resource, it's being applied in `deserializeIntoResource`. Any ideas on how to improve that?